binder/src/main/scala/PanamaCIRCTConverter.scala

```diff
@@ -574,13 +581,136 @@ class PanamaCIRCTConverter extends CIRCTConverter {
         .withNamedAttr("name", circt.mlirStringAttrGet(name))
         .withNamedAttr("nameKind", circt.firrtlAttrGetNameKind(FIRRTLNameKind.InterestingName))
         .withNamedAttr("annotations", circt.emptyArrayAttr)
-        .withNamedAttr("inner_sym", circt.hwInnerSymAttrGet(name))
         .withOperand(input)
         .withResult(util.convert(resultType))
         // .withResult( /* ref */ )
         .build()
+      firCtx.ops += ((id._id, op.op))
       firCtx.newItem(id, op.results(0))
     }
+
+    def emitConnect(dest: Reference.Value, srcVal: Reference.Value, loc: MlirLocation): Unit = {
+      val indexType = circt.mlirIntegerTypeGet(32)
+      var src = srcVal
+
+      // TODO: Strict connect
```

Comment:
match and throw unimplemented error

Reply:
The determination of strict connection may need adding something to FIRRTL C-API on the CIRCT side, I prefer just leave a comment here at the moment and fix it later.